packages/core/chrome/core-chrome-browser-internal/src/ui/header/header.tsx

```diff
@@ -90,8 +91,8 @@ export function Header({
   const isVisible = useObservable(observables.isVisible$, false);
   const [isNavOpen, setIsNavOpen] = useState(false);
   const [navId] = useState(htmlIdGenerator()());
-  const breadcrumbsAppendExtension = useObservable(breadcrumbsAppendExtension$);
 
+  const breadcrumbsAppendExtension = useObservable(breadcrumbsAppendExtension$);
```

Comment:
It looks like the newlines here were accidentally moved 😅 (I promise this is my last comment!)

```suggestion
const breadcrumbsAppendExtension = useObservable(breadcrumbsAppendExtension$);

```

Reply:
sweeet! 
ae8da0d thank you!!